src/components/ElementBar/ElementBar.test.tsx

```diff
@@ -87,16 +97,25 @@ describe('<ElementBar/>', () => {
     ).toBeInTheDocument();
   });
 
-  it('should delete an element', async () => {
+  it.each([
+    ['should delete a single element', ['element-1']],
+    ['should delete multiple elements', ['element-1', 'element-2']],
+  ])('%s', async (_testname, elementIds) => {
     const activeSlide = activeWhiteboardInstance.getSlide('slide-0');
+    activeSlide.setActiveElementIds(elementIds);
```

Comment:
It may be good to split it into several test cases, then expect part could be more explicit (of course some expressions will be duplicated). This is how it is done currently across the tests, but it is optional.

Reply:
As per discussion I want to prevent DRY here. Let's see how this is going.